app/models/region_kind.rb

```diff
@@ -0,0 +1,14 @@
+class RegionKind < ApplicationRecord
```

Comment:
This could've been named `RegionType`, but using the name `type` tends to conflict w/ Rails STI.  This is basically where we define the region hiearchy for each Simple instance.

Reply:
Do `RegionKind`s have to be in the database? I'm wondering if we can just initialize them using some config hash-map instead?

Feels like creating a table and db rows for this might be overkill.

Reply:
They do have to be in the DB, for a couple reasons:

* we need to know the canonical structure of Regions for building out the admin UI -- we could infer it from Region paths, but that would be error-prone and dangerous
* the names of Regions will vary between countries. For example, in India we have blocks, in Bangladesh we will have [upazila](https://en.wikipedia.org/wiki/Upazila).